project/Build.scala

```diff
@@ -88,7 +88,7 @@ object Build {
 
   val referenceVersion = "3.4.2-RC1"
 
-  val baseVersion = "3.6.0-RC1"
+  val baseVersion = "3.6.0"
```

Comment:
This change would break our current versioning of nightlies, which are defined using pattern `<baseVersion>-bin-YYYYMMDD-<commitSHA>-NIGHTLY` 
In theory all nightlies are always `-RC1` so there should be no conflicts with this one.
@Kordyjan any thoughts on that?

Reply:
we can always add the `-RC1` at the beginning of the nightlies' pattern instead of `baseVersion` if we *must* keep the same pattern : https://github.com/scala/scala3/blob/010ed5ab1f009e015b3e5436cea5cb1ac2f13340/project/Build.scala#L172